spatial-history/index.md

```diff
@@ -1,39 +1,33 @@
-  <button onclick="showMarkers(my_markers, my_map)" id="show"> show markers</button>
+
 </div>
   <div id="mapcontainer">
     <div id="map_canvas"></div>
   </div>
   <div id="map_legend"></div>
 </div>
 
-# Next Section
+<h2 id=history>History</h2>
+Looking at the creation of the Montreal video game cluster, it is clear that it began with the attraction of French gaming company Ubisoft to the city in 1997, as it would become one of the largest game publishers worldwide and it was after this that the influx of other large developers occurred. The draw for Ubisoft to Montreal was a series of policies financially incentivizing multimedia development in the city, particularly for French-language companies ([Pilon, pg 4](https://www.hindawi.com/journals/usr/2013/957630/)). Building off the success of Ubisoft Montreal and the creation of a number of quality assurance and other auxiliary companies, the city provided greater subsidies and professional training programs which, combined with the growing pool of skilled, creative game developers, established the city as a space for creators to find employment and for studios to have access to a larger selection of talented employees ([Grandadam, pg 1706](https://journals-scholarsportalinfo.myaccess.library.utoronto.ca/pdf/00343404/v47i0010/1701_psatdotvgiim.xml/)). These advantages caused the industry in Montreal to grow, but this growth became exponential as the city became enriched with third party companies able to provide necessary services for developers, allowing the entire development process to be carried out locally ([Darchen, pg 317](https://journals-scholarsportal-info.myaccess.library.utoronto.ca/pdf/09654313/v23i0002/311_pfccacgiimam.xml)). This produced the effect we can see on the map, which is a number of large-scale video game development clustered in a small radius, forming a cultural sphere of creative development intrinsically linked to the space it occupies.
+
+<h2 id=eco>Economic Significance</h2>
+
+The significance of the creation of this cluster from a spacial history perspective is threefold, the first significance being the economic impact it brought to the city. The nature of the video game development industry is such that when the necessary professionals and creative personnel can be found locally, a development studio will employ locally and disseminate the finished product globally, bringing a large amount of both jobs and capital into the city. The cluster and its economic boons haven’t shown signs of ending either, as EA began a 500 million dollar investment program into its Montreal studio Motive, pledging to create 500 new jobs in the city ([Sinclair, pg 1](https://www.gamesindustry.biz/articles/2017-04-07-ea-plans-to-add-500-new-jobs-in-montreal)). It also, for better or worse, changed the perception of the city’s job market such that it draws in talent from around the world, bringing in a large and diverse group of potential creators.
```

Comment:
SImilar to Internet tech companies in general, and in fact most kinds of corporate activity" goal is to form a centralized node which accumulates wealth simply by concentrating captial flows in a small region/small number of hands.